src/pkg/packager/sources/tarball.go

```diff
@@ -100,12 +105,14 @@ func (s *TarballSource) LoadPackage(ctx context.Context, dst *layout.PackagePath
 	if !dst.IsLegacyLayout() {
 		spinner := message.NewProgressSpinner("Validating full package checksums")
 		defer spinner.Stop()
+		l.Info("validating package checksums")
```

Comment:
Non-blocking: Is there an easy piece of metadata we can attach this to? Package name or source maybe?